DiscordBot.MessageHandlers/Helpers/RegexResponsesHelper.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Concurrent;
+using System.Text.RegularExpressions;
+
+namespace DiscordBot.MessageHandlers.Helpers
+{
+    public interface IRegexResponseHelper
+    {
+        bool IsMatch(string text);
+        string GetResponse(string text);
+    }
+
+    public class RegexResponsesHelper : IRegexResponseHelper
+    {
+        private readonly ConcurrentDictionary<string, string> regexResponses;
+
+        public RegexResponsesHelper() 
+        {
+            regexResponses = new ConcurrentDictionary<string, string>();
+
+            AddRegexResponse(@"((dzie([ń|n]) dobry)|(cze([s|ś][c|ć]))) .*", "Dzień dobry! ");
+            AddRegexResponse(@"dobranoc .*", "Dobranoc! ");
+            AddRegexResponse(@"wypierdalaj .*", "Sam wypierdalaj ");
+            AddRegexResponse(@"spierdalaj .*", "O ty chuju ");
+        }
+
+        private void AddRegexResponse(string key, string value)
```

Comment:
@Niewidzialny84 Why not just use `TryAdd`? https://learn.microsoft.com/en-us/dotnet/api/system.collections.concurrent.concurrentdictionary-2.tryadd?view=net-6.0

Reply:
Let it be future proof if we wanted to use more of them in database or something similar to responses and facts.

Reply:
In this case this should have name that suggest it also updates existing values